hello,py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/python
+def main():
+    print('Hi Sriram!!! just testing a pull request.')
+
+if __name__ == '__main__':
```

Comment:
Please explain the need for this condition

Reply:
This check is to prevent the code from executing directly if imported as a module. It will only execute via a python interpreter.